moveit_ros/perception/CMakeLists.txt

```diff
@@ -1,91 +1,83 @@
 cmake_minimum_required(VERSION 3.10.2)
 project(moveit_ros_perception)
 
-if(NOT "${CMAKE_CXX_STANDARD}")
-  set(CMAKE_CXX_STANDARD 14)
-endif()
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
+# Common cmake code applied to all moveit packages
+find_package(moveit_common REQUIRED)
+moveit_package()
 
 option(WITH_OPENGL "Build the parts that depend on OpenGL" ON)
 
-if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
-  set(CMAKE_BUILD_TYPE Release)
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
I think this should be in the `moveit_package.cmake` file inside moveit_common

Reply:
In the long run maybe, but I doubt all the packages will pass CI without changes to fix warnings if you add it there.  I think this is probably fine here for now and we should make an issue about what warning flags we want to be enabled by default and how and where we should specify them.

Reply:
Probably this should be addressed in another PR. Opened an issue #334 for this.